doc/classes/AudioStreamRandomPitch.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamRandomPitch" inherits="AudioStream" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Plays audio with random pitch tweaking.
 	</brief_description>
 	<description>
+		is used to make playback audio a bit more random.
```

Comment:
This is a very vague description. Something more like: "Randomly varies pitch during playback."